### PR TITLE
Manipulate  the humberger button with javascript

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
In this branch:
 I add script file
I manipulate the hamburger button using javascript

### CODE REVIEW

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
# Hi @TahenyBELGUITH 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
* Always try to make your commit message descriptive I can see your commit messages are not descriptive kindly fix it 🙏
* Kindly ensure you stick to the design as much as possible  
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 1]
```diff
@@ -8,76 +8,97 @@
...
+            <!-- Navigation links (hidden by default) -->
+            <a href=" ./index.html" class="active"><img class="toolbar-img2 " src="images/logo.svg " alt="Logo "></a>
+            <div id="myLinks" class="mylink links-desktop">
+                <a href="#portfolio">Portfolio</a>
+                <a href="#about">About</a>
+                <a href="#footer-form">Contact</a>
+            </div>
+            <!-- "Hamburger menu" / "Bar icon" to toggle the navigation links -->
+            <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                &#9776;
```
Comment (dasileker) [inline index.html:49]: - [ ] hello this function for your menu should contain also an `X` icon and target it to close [see here the specs](https://user-images.githubusercontent.com/61562626/128371981-d4de0a01-3cfb-42f0-b768-4480d05f88bb.jpeg)
-  it should contain an `X` after opening the menu [original](https://user-images.githubusercontent.com/61562626/128372965-ca9062c6-8377-41b5-997e-004e60f410a1.png) , and your is
btn1

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @TahenyBELGUITH** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -8,76 +8,122 @@
...
+                    </div>
+                    <ul>
+                        <li class="links"><a href="#portfolio">Portfolio</a>
+                        </li>
+                        <li class="links"><a href="#about">About</a>
+                        </li>
+                        <li class="links"><a href="#footer-form">Contact</a>
+                        </li>
+                    </ul>
+                </div>
```
Comment (mricanho) [inline index.html:68]: - [ ] [OPTIONAL] I'm gonna put this as optional, but it's really mandatory, when you open your website on a mobile device, the menu it's open by default, please change it to display: none by default, other than that your mobile menu works well.

[Check this link and open it on a mobile device](https://rawcdn.githack.com/TahenyBELGUITH/Portfolio-mobile-version/efba87ce29831814eb9862a814da4ac199bb07cc/index.html)

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @TahenyBELGUITH 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you, please fix the "optional" feature before your next milestone, I'm approving this milestone in order for you to not repeat a week, keep the good work👍🏽 

believe in change

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**